src/Layouts/Partials/Layer.vala

```diff
@@ -622,18 +622,18 @@ public class Akira.Layouts.Partials.Layer : Gtk.ListBoxRow {
     //     return false;
     // }
 
-    private void unselect_groups (Gtk.ListBox container) {
-        container.foreach (child => {
-            if (child is Akira.Layouts.Partials.Layer) {
-                Akira.Layouts.Partials.Layer layer = (Akira.Layouts.Partials.Layer) child;
+    //  private void unselect_groups (Gtk.ListBox container) {
+    //      container.foreach (child => {
+    //          if (child is Akira.Layouts.Partials.Layer) {
+    //              Akira.Layouts.Partials.Layer layer = (Akira.Layouts.Partials.Layer) child;
 
-                if (layer.grouped) {
-                    layer.container.unselect_all ();
-                    unselect_groups (layer.container);
-                }
-            }
-        });
-    }
+    //              if (layer.grouped) {
+    //                  layer.container.unselect_all ();
+    //                  unselect_groups (layer.container);
+    //              }
+    //          }
+    //      });
+    //  }
 
```

Comment:
Not "normal" per se, but it's okay since this still an alpha and not production ready.
I comment that method because it's not used yet and it was logging an error in the debug.